core/providers/anthropic/chat.go

```diff
@@ -310,63 +317,55 @@ func (response *AnthropicMessageResponse) ToBifrostChatResponse() *schemas.Bifro
 
 	// Process content and tool calls
 	if response.Content != nil {
-		if len(response.Content) == 1 && response.Content[0].Type == AnthropicContentBlockTypeText {
```

Comment:
there wont be cache control blocks in response ever so you can remove it everywhere